client/src/components/Ticketing/ticketingmanager/Refund/RefundOrderTicket.tsx

```diff
@@ -1,42 +1,174 @@
-        <div className='flex flex-row'>
-          <h1 className='font-bold text-5xl bg-clip-text text-transparent bg-gradient-to-r from-sky-500 to-indigo-500 mb-14'>Refund Ticket</h1>
-        </div>
-        <div className='flex flex-wrap justify-center h-fit gap-x-24'>
-          <div className='w-80 bg-gradient-to-r from-sky-500 to-indigo-500 drop-shadow-xl
-            text-center rounded-xl m-3 p-3'>
-            <h2 className='text-xl font-bold text-white m-4'>
-              Stripe Order
-            </h2>
-            <div className='flex flex-wrap w-64 mx-auto'>
-              <label htmlFor='payment-intent' className='text-white text-l' id='payment-input'>
-                Payment Intent
-              <input className='text-black w-full mt-1 mb-3 px-2 py-2 rounded' type='text'
-                id='payment-intent' name='payment-intent'
-                placeholder='XXXXX...'></input>
-              </label>
-            </div>
-            <div className='flex flex-wrap w-64 mx-auto'>
-              <label htmlFor='refund-intent' className='text-white text-l' id='refund-input'>
-                Refund Intent
-              <input className='text-black w-full mt-1 mb-3 px-2 py-2 rounded' type='text'
-                id='refund-intent' name='refund-intent'
-                placeholder='XXXXX...'></input>
-              </label>
+  const {token} = useFetchToken();
+  const [show, setShow] = useState({
+    showPopUp: false,
+    message: '',
+    title: '',
+    success: true,
+    handle: () => {
+      setShow((p) => ({...p, showPopUp: false}));
+    },
+  });
+  const onSubmit = async (orderID) => {
+    try {
+      const response = await fetch(
+        `${process.env.REACT_APP_API_2_URL}/order/refund/${orderID}`,
+        {
+          credentials: 'include',
+          method: 'put',
+          headers: {
+            'Content-Type': 'application/json',
+            'Authorization': `Bearer ${token}`,
+          },
+        },
+      );
+
+      if (!response.ok) {
+        throw response;
+      }
+
+      setShow({
+        ...show,
+        showPopUp: true,
+        message: `Order ${orderID} successfully refunded`,
+        title: `Order Refund Successful`,
+        success: true,
+      });
+    } catch (error) {
+      const errorMessage = await error.json();
+      setShow({
+        ...show,
+        showPopUp: true,
+        message: `Order ${orderID} refund unsuccessful ${errorMessage.error}`,
+        title: `Order Refund Failed`,
+        success: false,
+      });
+    }
+  };
+  const mappedInstances = [
+    {
+      name: 'John Doe',
+      date: '2023-10-24',
+      showings: ['Gone With The Wind', 'Sound Of Music'],
+      orderID: '12345',
+      price: 19.99,
+    },
+    {
+      name: 'John Doe',
+      date: '2023-10-25',
+      showings: 'Gone With The Wind',
+      orderID: '12345',
+      price: 14.99,
+    },
+  ];
```

Comment:
Is this dummy data that will be removed later?

Reply:
Correct, dummy data will be removed since this is just front end development